jobs/wsCleanupJob.groovy

```diff
@@ -0,0 +1,30 @@
+String listOfSlaves(int start, int end) {
+  String slaveList = ""
+  for (i = start; i <= end; i++) {
+    slaveList = slaveList + "rhsm-jenkins-slave${i}"
+    if ( i < end ) {
+      slaveList = slaveList + ","
+    }
+  }
+  return slaveList
+}
+
+job("WsCleanup"){
+    description('This job deletes leftover ws-cleanup files across all the slave nodes.')
+    label('rhsm')
+    parameters {
+        stringParam('JENKINS_SLAVES', listOfSlaves(0,3), 'Hostnames of servers to run on.')
```

Comment:
```suggestion
        labelParam('NODE_LABEL') {
            defaultValue('rhsm')
            description('Select nodes')
            allNodes('allCases', 'IgnoreOfflineNodeEligibility')
        }
```
This will work for all nodes that are online, since we no longer have static slaves 0..3.    With that change you can remove the `wrappes { sshAgent() }`,  the jssh logic in the bash script (since it'll just run locally) and the listOfSlaves().  We should probably make this change to our other docker cleanup job since this feature exists now.

Reply:
Done.